tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -257,6 +257,14 @@ static int test_wdt_callback_1(void)
 		}
 	}
 
+	err = wdt_disable(wdt);
+	if (err < 0) {
+		if (err != -EPERM) {
+			TC_PRINT("Watchdog disable error\n");
+			return TC_FAIL;
+		}
+	}
```

Comment:
Can we simplify this a bit?

```suggestion
	err = wdt_disable(wdt);
	if (err < 0 && err != -EPERM) {
		TC_PRINT("Watchdog disable error\n");
		return TC_FAIL;
	}
```

Reply:
Done!